Hydra/Serializer/CollectionFiltersEnhancer.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Hydra\Serializer;
+
+use Dunglas\ApiBundle\Api\Filter\FilterInterface;
+use Dunglas\ApiBundle\Api\ResourceInterface;
+use Dunglas\ApiBundle\Api\ResourceResolver;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\SerializerAwareNormalizer;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * Enhance the result of collection by adding the filters applied on collection.
+ *
+ * @author Samuel ROZE <samuel.roze@gmail.com>
+ */
+class CollectionFiltersEnhancer extends SerializerAwareNormalizer implements NormalizerInterface
```

Comment:
What about `CollectionFiltersDecorator` instead of `Enhancer`? Would reflect better what this is IMO.